proto/minder/v1/minder.proto

```diff
@@ -170,7 +170,13 @@ service ArtifactService {
 message ListArtifactsRequest {
     string provider = 1 [deprecated=true];
     Context context = 3;
-    string from = 4;
+    string from = 4 [
+        (buf.validate.field).string = {
+            pattern: "^[^=]+=[\\S]+$",
```

Comment:
I think you want to be a little more prescriptive here; this pattern allows `"    =="`.

It looks like the left-hand-side is a C-style identifier (`[:word:]`, and I'm not sure what the right-hand side should be, but I'd expect it to be some sort of word-like thing (it's called repoSlubList in our current code: https://github.com/mindersec/minder/blob/http-send/internal/controlplane/handlers_artifacts.go#L203

Maybe:

`"^[[:word:]]+=[-,/[:word:]]+$"`

(That's "c-identifier" followed by hyphen+slash+comma-separated c-identifiers)